test/utilities_test.jl

```diff
@@ -8,4 +8,6 @@
     @test PkgAuthentication.assert_dict_keys(Dict("bar" => 0), "foo", "bar"; msg="") === nothing
     @test_throws ErrorException PkgAuthentication.assert_dict_keys(Dict(), "foo", "bar"; msg="")
     @test_throws ErrorException PkgAuthentication.assert_dict_keys(Dict("baz" => 0), "foo", "bar"; msg="")
+
+    @test PkgAuthentication.detectwsl() isa Bool
```

Comment:
The test is a bit contrived, but this should at least exercise the code path now.